package/MDAnalysis/analysis/rdf.py

```diff
@@ -229,6 +234,13 @@ def __init__(self,
         self.rdf_settings = {'bins': nbins,
                              'range': range}
         self._exclusion_block = exclusion_block
+        if exclude_same is not None and exclude_same not in ['residue', 'segment', 'chain']:
+            raise ValueError(
+                "The exclude_same argument to InterRDF must be None, 'residue', 'segment' "
+                "or 'chain'."
+            )
+        name_to_attr = {'residue': 'resindices', 'segment': 'segindices', 'chain': 'chainids'}
```

Comment:
Does this have to be something else? chainids didnt seem to work for me but could be messing something up.

Reply:
I think it's actually chainIDs, not chainids, fixed in my most recent commits.